packaging/fedora/build

```diff
@@ -20,6 +20,7 @@ import time
 CH_BASE = os.path.abspath(os.path.dirname(__file__) + "/../..")
 CH_RUN  = [CH_BASE + "/bin/ch-run"]
 PACKAGES = ["charliecloud",
+            "charliecloud-builder", # depends on -doc
```

Comment:
Why does it depend on `-doc`? If it's some Python thing I wonder if there is a better way to do it.

Reply:
It doesn't, this is left over crud. Good catch.